<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISChess</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.png</normaloff>assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="chessboardView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="settingsPanel">
      <item>
       <widget class="QGroupBox" name="boardGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Board</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="currentBoard">
           <item>
            <widget class="QLabel" name="currentBoardLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Current:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="currentBoardValue">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="boardActions">
           <item row="0" column="0">
            <widget class="QPushButton" name="loadBoard">
             <property name="toolTip">
              <string>Load a board from a file</string>
             </property>
             <property name="statusTip">
              <string>Load a board from a file</string>
             </property>
             <property name="text">
              <string>&amp;Load</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="copyBoard">
             <property name="toolTip">
              <string>Copy the board as FEN</string>
             </property>
             <property name="statusTip">
              <string>Copy the board as FEN</string>
             </property>
             <property name="text">
              <string>&amp;Copy</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="exportBoard">
             <property name="toolTip">
              <string>Export the board to a file</string>
             </property>
             <property name="statusTip">
              <string>Export the board to a file</string>
             </property>
             <property name="text">
              <string>&amp;Export</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="reloadBoard">
             <property name="toolTip">
              <string>Reload the current board</string>
             </property>
             <property name="statusTip">
              <string>Reload the current board</string>
             </property>
             <property name="text">
              <string>&amp;Reload</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="botsGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Bots</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QScrollArea" name="botsScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="botsList">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>254</width>
              <height>18</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gameGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Game</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="moveActions">
           <item>
            <widget class="QPushButton" name="prevMove">
             <property name="toolTip">
              <string>Undo the last move</string>
             </property>
             <property name="statusTip">
              <string>Undo the last move</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="edit-undo">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startStop">
             <property name="toolTip">
              <string>Start/Stop the game</string>
             </property>
             <property name="statusTip">
              <string>Start/Stop the game</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="media-playback-start">
               <normaloff>../../../.designer/backup</normaloff>../../../.designer/backup</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextMove">
             <property name="toolTip">
              <string>Redo the next move</string>
             </property>
             <property name="statusTip">
              <string>Redo the next move</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="edit-redo">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="autoMoves">
           <item>
            <widget class="QLabel" name="autoMovesLabel">
             <property name="text">
              <string>Moves:</string>
             </property>
             <property name="buddy">
              <cstring>autoMovesCount</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="autoMovesCount"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="movesSep">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="movesList">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>64</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>100</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>#</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Move</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Player</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBoard">
    <property name="title">
     <string>&amp;Board</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionReload"/>
    <addaction name="actionCopy"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuBoard"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="icon">
    <iconset theme="SP_DirOpenIcon"/>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="toolTip">
    <string>Load a board from a file</string>
   </property>
   <property name="statusTip">
    <string>Load a board from a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload the current board</string>
   </property>
   <property name="statusTip">
    <string>Reload the current board</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy the board as FEN</string>
   </property>
   <property name="statusTip">
    <string>Copy the board as FEN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset theme="SP_DriveFDIcon"/>
   </property>
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="toolTip">
    <string>Export the board to a file</string>
   </property>
   <property name="statusTip">
    <string>Export the board to a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>&amp;Start/Stop</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop the game</string>
   </property>
   <property name="statusTip">
    <string>Start/Stop the game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo the last move</string>
   </property>
   <property name="statusTip">
    <string>Undo the last move</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo the next move</string>
   </property>
   <property name="statusTip">
    <string>Redo the next move</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>chessboardView</tabstop>
  <tabstop>loadBoard</tabstop>
  <tabstop>reloadBoard</tabstop>
  <tabstop>copyBoard</tabstop>
  <tabstop>exportBoard</tabstop>
  <tabstop>botsScrollArea</tabstop>
  <tabstop>prevMove</tabstop>
  <tabstop>startStop</tabstop>
  <tabstop>nextMove</tabstop>
  <tabstop>autoMovesCount</tabstop>
  <tabstop>movesList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>loadBoard</sender>
   <signal>clicked()</signal>
   <receiver>actionLoad</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportBoard</sender>
   <signal>clicked()</signal>
   <receiver>actionExport</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyBoard</sender>
   <signal>clicked()</signal>
   <receiver>actionCopy</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reloadBoard</sender>
   <signal>clicked()</signal>
   <receiver>actionReload</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextMove</sender>
   <signal>clicked()</signal>
   <receiver>actionRedo</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>884</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startStop</sender>
   <signal>clicked()</signal>
   <receiver>actionStart</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevMove</sender>
   <signal>clicked()</signal>
   <receiver>actionUndo</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
